_includes/author.html

```diff
@@ -0,0 +1,28 @@
+{%- assign author = include.author %}
+{%- if author.id %}
+  {%- assign name = nil %}
+  {%- assign link = nil %}
+  {%- assign affiliation = nil %}
+  {%- for group in site.data.organization-members %}
+    {%- for member in group.members %}
+      {%- if author.id == member.id %}
+        {%- assign name = member.name %}
+        {%- assign link = "/o-nas/#" | append: member.id %}
+        {%- if author.show_internal_affiliation %}
+          {%- assign affiliation = site.title %}
+        {%- endif %}
+      {%- endif %}
+    {%- endfor %}
+  {%- endfor %}
+{%- else %}
+  {%- assign name = author.name %}
+  {%- assign link = author.link %}
+  {%- assign affiliation = author.affiliation %}
+{%- endif %}
+
+{%- if link %}
+  <a href="{{ link }}"><span class="name">{{ name }}</span></a>
+{%- else %}
+  <span class="name">{{ name }}</span>
+{%- endif %}
```

Comment:
I think it would be useful to include an error case here, such as when `name` is null? I suppose that could happen when there's a typo in the `id` or the author is not a member.

Reply:
Make sense (was slightly more complicated in the current version but it's worth it). Done.